src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/Native/Interop.crypt32.cs

```diff
@@ -314,9 +314,6 @@ public static bool CertVerifyCertificateChainPolicy(ChainPolicy pszPolicyOID, Sa
         [DllImport(Libraries.Crypt32, CharSet = CharSet.Unicode, SetLastError = true)]
         public static extern bool CertFreeCertificateContext(IntPtr pCertContext);
 
-        [DllImport(Libraries.Crypt32, CharSet = CharSet.Unicode, SetLastError = true)]
-        public static extern bool CertCloseStore(IntPtr hCertStore, int dwFlags);
```

Comment:
This version used a `int` instead of a `uint`. I don't think there is an impact for this code specifically, I prefered the uint because this is [what the docs suggest](https://docs.microsoft.com/en-us/windows/desktop/api/wincrypt/nf-wincrypt-certclosestore) (apparently that param is a `DWORD`)